aws/app/lambda/reliability/lib/markdown.js

```diff
@@ -16,8 +16,8 @@ module.exports = (formResponse, language) => {
 
   // use Notify lang attribute to denote the language https://notification.canada.ca/format
   const emailBody = language === "fr" ?
-      "[[fr]]\n" + json2md([{ h1: title }, mdBody])+ "\n[[/fr]]":
-      "[[en]]\n" + json2md([{ h1: title }, mdBody])+ "\n[[/en]]";
+      "[[fr]]\n" + json2md([{ h1: title }, { h5: "Date" }, { p: submissionDateTime }, mdBody])+ "\n[[/fr]]":
```

Comment:
If we're now using a timestamp as opposed to a string then it should be converted to an ISO string here. A normal user is not going to understand a unix timestamp in their email